package.json

```diff
@@ -258,14 +258,14 @@
           "swift.disableAutoResolve": {
             "type": "boolean",
             "default": false,
-            "markdownDescription": "Disable automatic running of `swift package resolve`. This will also disable searching for command plugins and the initial test discovery process as both of these can kick of a resolve.",
+            "markdownDescription": "Disable automatic running of `swift package resolve` whenever the `Package.swift` or `Package.resolve` files are updated. This will also disable searching for command plugins and the initial test discovery process.",
             "scope": "machine-overridable",
             "order": 7
           },
           "swift.diagnosticsCollection": {
             "type": "string",
             "default": "keepSourceKit",
-            "description": "Controls how diagnostics from the various providers are merged into the `swift` diagnostics collection.",
+            "markdownDescription": "Controls how diagnostics from the various providers are merged into the `swift` diagnostics collection.",
```

Comment:
Should probably make it clear this is related to diagnostics used in the problems pane. The word diagnostic is not used in the UI anywhere